.github/workflows/ci.yml

```diff
@@ -132,19 +136,27 @@ jobs:
           echo "::set-output name=deployment-matrix::$MATRIX"
 
   build-image-amd64:
-    uses: ./.github/workflows/build-docker-image-singlearch.yml
+    uses: ./.github/workflows/build-docker-image.yml
     needs: [initialize-matrix]
     with:
       platform: amd64
       deployment-matrix-str: ${{ needs.initialize-matrix.outputs.deployment-matrix }}
 
   build-image-aarch64:
-    uses: ./.github/workflows/build-docker-image-singlearch.yml
+    uses: ./.github/workflows/build-docker-image.yml
     needs: [initialize-matrix]
     with:
       platform: aarch64
       deployment-matrix-str: ${{ needs.initialize-matrix.outputs.deployment-matrix }}
 
+  build-image-multiarch:
+    uses: ./.github/workflows/build-docker-image.yml
+    needs: [initialize-matrix]
+    with:
+      platform: multiarch
+      multiarch: true
+      deployment-matrix-str: ${{ needs.initialize-matrix.outputs.deployment-matrix }}
+
```

Comment:
why are you building the images again? amd64 and aarch64 are already build in the steps before.

Reply:
I didn't find a way to merge the multiarech images from archive files (tar), it is only possible with Skopeo to merge images from different container registries.

Reply:
and what is the problem with skopeo? I mean @doosuu split the container-build to reduce the build-time and now we are moving a step back.

Reply:
Correction:
Sorry, I meant using **"docker buildx imagetools create**" not skopeo.
Generally, buildx tools can merge multi-single architecture images to single multi-arch image but they have to be deployed to in container registries.